.github/workflows/init-release.yaml

```diff
@@ -74,4 +74,25 @@ jobs:
           signoff: true
           labels: release
 
+  trigger-release:
```

Comment:
It should be executed after init-release PR is merged

Reply:
I have created a new workflow file. PTAL!